x/bounty/client/cli/utils.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/tendermint/tendermint/libs/tempfile"
+	"os"
+)
+
+// SaveKey saves the given key to a file as json and panics on error.
+func SaveKeys(pubKey cryptotypes.PubKey, privKey cryptotypes.PrivKey, dirPath string, cdc codec.Codec) {
```

Comment:
Is this a formal version about store key to local? The `public key` is not used and the `private key` is saved to a 666 permission file which may be seen by other users.

Reply:
This is just a placeholder to store the decryption key for the bounty program creator, and yes it should be much better dealing with the keys/permissions.